README.md

```diff
@@ -5,6 +5,15 @@ A Slicer extension that enables user to interact with a Slicer scene using virtu
 
 ![](VR.png)
 
+Building steps
+--------------
+
+- Clone Slicer from here: https://github.com/jbvimort/Slicer/tree/VR
+- Checkout on the VR branch
+- Build Slicer thanks to a script similar to this one: https://gist.github.com/jbvimort/903f626b27587a2a137293da0d0cf5e0 (take care to modify the path to adapt it to your environment)
+- Build the extension against the newly built Slicer
+- If you want to run the extension in Slicer by adding the path to its build tree: you need to use the command option  --launcher-additional-settings and specifying the path to the Build\inner-build\AdditionalLauncherSettings.ini file
+
```

Comment:
- Clone Slicer from https://github.com/jbvimort/Slicer/tree/VR and checkout [VR](https://github.com/jbvimort/Slicer/tree/VR) branch

    ```
    git clone https://github.com/jbvimort/Slicer -b VR
    ```

- Build Slicer using a script similar to this one: https://gist.github.com/jbvimort/903f626b27587a2a137293da0d0cf5e0 (take care to modify the path to adapt it to your environment)

- Build the extension against the newly built Slicer

- To start Slicer from a build tree and ensure the extension is properly loaded, considering using the ``--launcher-additional-settings`` option:

    ```
    ./Slicer.exe  --launcher-additional-settings C:\path\to\SlicerOpenVR-build\inner-build\AdditionalLauncherSettings.ini --additional-module-paths C:\path\to\SlicerOpenVR-build\inner-build\
    ```

Note that specifying the top-level build directory of the extension ensures that Slicer find all types of modules.